specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/rbac.json

```diff
@@ -27,38 +27,38 @@
     "application/json"
   ],
   "paths": {
-    "/{scope}/providers/Microsoft.Authorization/roleDefinitions": {
-      "get": {
+        "/{scope}/providers/Microsoft.Authorization/roleDefinitions/{roleDefinitionName}": {
+      "delete": {
         "tags": [
           "RoleDefinitions"
         ],
-        "operationId": "RoleDefinitions_List",
-        "description": "Get all role definitions that are applicable at scope and above.",
+        "operationId": "RoleDefinitions_Delete",
+        "description": "Deletes a custom role definition.",
```

Comment:
Question: should an error be expected if someone tries to delete a built-in role definition?

Reply:
yes